From 2a98ffac7733e77a178af7240f9fa9392ca45aad Mon Sep 17 00:00:00 2001
From: Cihan Karadag <cihan.cihan@gmail.com>
Date: Sun, 5 Jul 2026 15:53:40 -0600
Subject: [PATCH 06069/15333] selftests/rseq: Add missing test binaries to
 .gitignore

legacy_check and check_optimized are built by this Makefile via
TEST_GEN_PROGS_EXTENDED but are not listed in .gitignore, causing them
to show up as untracked files after a build.

Signed-off-by: Cihan Karadag <cihan.cihan@gmail.com>
Signed-off-by: Thomas Gleixner <tglx@kernel.org>
Link: https://patch.msgid.link/20260705215346.1132764-1-cihan.cihan@gmail.com
---
 tools/testing/selftests/rseq/.gitignore | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/tools/testing/selftests/rseq/.gitignore b/tools/testing/selftests/rseq/.gitignore
index ec01d164c1f0..9b6eb6c3922f 100644
--- a/tools/testing/selftests/rseq/.gitignore
+++ b/tools/testing/selftests/rseq/.gitignore
@@ -11,3 +11,5 @@ param_test_mm_cid_benchmark
 param_test_mm_cid_compare_twice
 syscall_errors_test
 slice_test
+legacy_check
+check_optimized
-- 
2.55.0

